src/Microsoft.AspNetCore.Hosting.Server.Abstractions/IServer.cs

```diff
@@ -19,7 +19,8 @@ public interface IServer : IDisposable
         /// <summary>
         /// Start the server with an HttpApplication.
         /// </summary>
-        /// <param name="application">An instance of <see cref="IHttpApplication"/>.</param>
+        /// <param name="application">An instance of <see cref="IHttpApplication{TContext}"/>.</param>
+        /// <typeparam name="TContext">The context associated with the HttpApplication.</typeparam>
```

Comment:
I just used the same wording we have already used https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Hosting.Server.Abstractions/IHttpApplication.cs#L11 and https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Hosting.Server.Abstractions/IServer.cs#L20.
But there is no `HttpApplication` as such.